tools/build.py

```diff
@@ -0,0 +1,150 @@
+#
+# Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+#
+
+import argparse
+import os
+import shutil
+import subprocess
+import sys
+
+if __name__ == "__main__":
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--clone_dir", required=True)
+    parser.add_argument("--conda_path")
+    parser.add_argument("--config_mode", default="Shipping")
+    parser.add_argument("--unreal_engine_dir", required=True)
+    parser.add_argument("--num_parallel_jobs", type=int, default=1)
+    parser.add_argument("--target_platform", required=True, help="Values accepted are Win64, Linux, Mac")
+    parser.add_argument("--output_dir", required=True)
+    parser.add_argument("--tag", required=True)
+    args = parser.parse_args()
+
+    assert args.target_platform in ["Linux", "Mac"] and args.conda_path
+
+    # clone repo along with submodules
+    cmd = ["git", "clone", "--recurse-submodules", "https://github.com/isl-org/spear", args.clone_dir]
+    print(f"[SPEAR | build.py] Executing: {' '.join(cmd)}")
+    cmd_result = subprocess.run(cmd)
+    assert cmd_result.returncode == 0
+
+    os.makedirs(args.output_dir, exist_ok=True)
+
+    # change working directory to the cloned directory above
+    cwd = os.getcwd()
+    print(f"[SPEAR | build.py] Changing current working dir from {cwd} to {args.clone_dir}")
+    os.chdir(args.clone_dir)
+
+    # create a new virtual python env called spear-env
```

Comment:
- call this script package_executable.py
- don't call the environment spear-env because that is what I usually call it on my machine. Call it spear-package-executable-tmp-env

Reply:
Agreed, changed.